code/game/objects/effects/poster_motivational.dm

```diff
@@ -0,0 +1,291 @@
+#define POSTER_MOOD_CAT "poster_mood"
+
+/obj/item/poster/quirk
+	name = "placeholder quirk poster"
+	desc = "Uh oh! You shouldn't have this!"
+	icon_state = "rolled_poster_legit"
+	var/quirk_poster_department = NONE
+
+/obj/item/poster/quirk/Initialize(mapload, obj/structure/sign/poster/new_poster_structure)
+	. = ..()
+
+	register_context()
+
+	if(poster_structure)
+		var/obj/structure/sign/poster/quirk/department_grab = poster_structure
+		department_grab.quirk_poster_department = quirk_poster_department
+
+/// You can use any spraypaint can on a quirk poster to turn it into a contraband poster from the traitor objective
+/obj/item/poster/quirk/attackby(obj/item/postertool, mob/user, params)
+	if(!is_special_character(user))
+		return ..()
+	if(!HAS_TRAIT(user, TRAIT_POSTERBOY))
+		return ..()
+	if(!istype(postertool, /obj/item/toy/crayon))
+		return ..()
+	balloon_alert(user, "converting poster...")
+	if(!do_after(user, 5 SECONDS, user))
+		balloon_alert(user, "interrupted!")
+		return
+	var/obj/item/poster/traitor/quirkspawn = new(get_turf(src))
+	user.put_in_hands(quirkspawn)
+	to_chat(user, span_notice("You have converted one of your posters!"))
+	qdel(src)
+
+/// Screentip for the above
+
+/obj/item/poster/quirk/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	if(!is_special_character(user))
+		return NONE
+	if(!HAS_TRAIT(user, TRAIT_POSTERBOY))
+		return NONE
+	if(!istype(held_item, /obj/item/toy/crayon))
+		return NONE
+	context[SCREENTIP_CONTEXT_LMB] = "Turn into Demoralizing Poster"
+	return CONTEXTUAL_SCREENTIP_SET
+
+/obj/structure/sign/poster/quirk
+	name = "quirk poster"
+	desc = "A large piece of homemade space-resistant printed paper."
+	poster_item_name = "homemade poster"
+	poster_item_desc = "A poster made with love, some people will enjoy seeing it."
+	poster_item_icon_state = "rolled_legit"
+	var/quirk_poster_department = NONE
+	var/datum/proximity_monitor/advanced/quirk_posters/mood_buff
+
+/obj/structure/sign/poster/quirk/on_placed_poster(mob/user)
+	mood_buff = new(src, 7, TRUE, quirk_poster_department)
```

Comment:
Name your args. https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#prefer-named-arguments-when-the-meaning-is-not-obvious